internal/acceptance/metastore_data_access_test.go

```diff
@@ -4,22 +4,22 @@ import (
 	"testing"
 )
 
-func TestUcAccMetastoreDataAccessOnAws(t *testing.T) {
```

Comment:
Shouldn't we keep this test to ensure we're covering workspace-level operations?

Reply:
added test back